source/tomopy/prep/stripe.py

```diff
@@ -679,6 +679,7 @@ def _detect_stripe(listdata, snr):
         xlist[ndrop:-ndrop - 1], listsorted[ndrop:-ndrop - 1], 1)
     numt1 = _intercept + _slope * xlist[-1]
     noiselevel = np.abs(numt1 - _intercept)
+    noiselevel = np.clip(noiselevel, 1e-6, None)
```

Comment:
Is this clipping operation related to the zero-division?

Reply:
Yes, it is. This is a precaution for the case that the input is noise-free (e.g simulation data).